miio/vacuum_cli.py

```diff
@@ -557,6 +561,25 @@ def carpet_mode(vac: miio.Vacuum, enabled=None):
         click.echo(vac.set_carpet_mode(enabled))
 
 
+@cli.command()
+@click.argument("mode", required=False, type=str)
+@pass_dev
+def carpet_cleaning_mode(vac: miio.Vacuum, mode=None):
+    """Query or set the carpet cleaning/avoidance mode.
+
+    Allowed values: Avoid, Rise, Ignore
+    """
+    from miio.vacuum import CarpetCleaningMode
+
+    if mode is None:
+        click.echo("Carpet cleaning mode: %s" % vac.carpet_cleaning_mode().name)
```

Comment:
```suggestion
        click.echo("Carpet cleaning mode: %s" % vac.carpet_cleaning_mode())
```
Better simply use the direct response here, I didn't check if this can return none or not, but in case of none this would cause an exception.